WordPress/Classes/ViewRelated/Comments/CommentModerationBar.swift

```diff
@@ -67,6 +67,14 @@ class CommentModerationBar: UIView {
         NotificationCenter.default.removeObserver(self)
     }
 
+    override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        // readjust stack view width when horizontal size class changes.
```

Comment:
Shouldn't we calling the super here?
`super.traitCollectionDidChange(previousTrailtCollection)`